test/plugins/file-description-task-description-plugin-test.js

```diff
@@ -0,0 +1,48 @@
+const buster = require('buster')
+const plugin = require('../../lib/plugins/file-description-task-description-plugin.js')
```

Comment:
Teste, dass das plugin compareTwoStrings mit den gegebenen Strings aufruft und zurück gibt, was es von compareTwoStrings bekommt. Ansonsten ändert string-similarity in der nächsten Version vielleicht den Algorithmus leicht und es kommt nicht mehr 0.75 sondern 0.79 raus und alle tests schlagen fehl.